misc/scripts/validate_extension_api.sh

```diff
@@ -70,9 +70,9 @@ while read -r file; do
     new_validation_error="$(comm -23 "$validation_output" "$allowed_errors")"
     obsolete_validation_error="$(comm -13 "$validation_output" "$allowed_errors")"
 
-    if [ -n "$obsolete_validation_error" ] && [ "$warn_extra" = "1" ]; then
-        make_annotation "The following validation errors no longer occur (compared to $reference_tag):" "$obsolete_validation_error" warning "$file"
-    fi
+    #if [ -n "$obsolete_validation_error" ] && [ "$warn_extra" = "1" ]; then
+    #    make_annotation "The following validation errors no longer occur (compared to $reference_tag):" "$obsolete_validation_error" warning "$file"
```

Comment:
I guess we could still print instead of using `make_annotation`, so they still appear in logs but don't clutter the PRs with annotations.

Reply:
Alright! In my latest push, it'll just print the message rather than make an annotation.